docs/release-notes/mia-platform-v13.mdx

```diff
@@ -7,3 +7,140 @@ image: "img/release-note-link-preview.png"
+Many are the benefits of adopting the Fast Data control plan in your Fast Data Projects, for example:
+
+- **Optimize Resource Allocation**, by strategically pausing and resuming the consumption of data streams along the Fast Data pipelines
+- **Initial Load and Full Refresh processes management**, thanks to a user-friendly interface and without altering the configuration of your workloads deployed in runtime 
+- **Performance Testing and Simulation**, to simulate different scenarios for performance testing by observing your system behavior under various conditions
+- **Enhanced System Reliability**, allowing you to gracefully handle unexpected situations and/or system maintenance periods while minimizing system downtime.
+
+To learn more, please refer to the [documentation](link).
+
+### Streamlined how you build your Saga, now No-Code
+
+Let's welcome a new brand section: the Flow Manager Configurator!
+
+ADD IMAGE  
+
+Streamline your workflow with an intuitive graphical interface, designed to simplify the configuration of saga flows for Flow Manager services.  
+Enjoy no-code editing and seamless service linking, all compatible with Flow Manager versions 2.0.0 and up.  
+Explore [our documentation](link) for more details.
+
+### Blueprint and Template management via User Interface
+
+Managing the company's Blueprint with their respective Templates and settings is now a child'd play: with a dedicated interface, Templates, Environments, and Workload & Runtime settings for all new Company Projects can be created, set up, and modified in one click.
+
+Forget about using the CMS and use the Console as the star center to manage and govern the entire project lifecycle!
+
+ADD IMAGE
+
+To learn more, please refer to the [documentation](link).
+
+### Manage Sidecar containers with Mia-Platform Console
+
+Governing Sidecar containers in your Projects has never been so simple: in the Design area of Mia-Platform Console, we have introduced a brand new `Sidecars` section where you can view and manage all the configurations of your Sidecar containers. You can also manage those Sidecar configurations in the detail page of each of your Microservices.
+To discover more about how to configure and manage Sidecars, visit the related documentation page.
+
+ADD LINK
+ADD IMAGE
+
+## Conversational DevX Powered by AI
+
+This version of Mia-Platform Console is the first version powered by Artificial Intelligence! Welcome to our new Mia-Assistant that will support Mia-Platform users in speeding up onboarding and accelerating development.
+
+The main benefits for the user can be summarized by the following:
+
+- Speed up the onboarding when discovering our Mia-Platform Console
+- Facilitates understanding of product features
+- Simplify product discoverability
+- Speed up the doc search
+- Action Advisory
+- Simplify access to features
+
+Let's go in deep with the new Features:
+
+### Mia-Assistant
```

Comment:
Let's say that it is released in 'experiment' status, explaining what it means

Reply:
I wrote in the last line that it is in "Experimental" phase. I think that it's quite self-explanatory. Or do you think that we should make it more explicit? In that case, have you got an alternative proposal?